packages/tools/kadena-cli/src/prompts/network.ts

```diff
@@ -295,17 +295,20 @@ export const networkDeletePrompt: IPrompt<string> = async (
   }
 
   let message = `Are you sure you want to delete the configuration for network "${defaultValue}"?`;
-  if (previousQuestions.isDefaultNetwork === true) {
-    message += `\nYou have currently set this as your "default network". If you delete it, then the default network settings will also be deleted.`;
+  if (isNotEmptyString(previousQuestions.isDefaultNetwork)) {
+    message += `\nThis is a "default network". If you delete it, then the default network settings will also be deleted.`;
```

Comment:
i guess this change coming from merge conflict. as part of this PR https://github.com/kadena-community/kadena.js/pull/1954/files#diff-66305d0f5df0f0472b98ec61b2dc0a81e2bfb3858044d61e8f567d35f6cc2f0dL298-L300 updated it